InteractionBase/src/commands/Command.h

```diff
@@ -39,6 +39,7 @@ namespace Interaction {
 class INTERACTIONBASE_API Command
 {
 	public:
+        Command(bool canBeUsedInMenu = false, QString name = "");
```

Comment:
So, I'm a bit uneasy that name has a default argument which is empty. I realize that you're just trying to make things work as they are, but logically, every command (menu or not) has a name, and it would make more sense to change this so that all existing uses of Command provide a name to the constructor. In that case the name argument should come first (actually it should come first anyway, as it is more important than canBeUsedInMenu) and not have a default value.

Also note however, that a name is not the same as a label/visualization. Actually, I'm wondering if we need a name at all. Perhaps we just need something that give us a menu entry for the command. Menu entry being a visualization.